pipelines/rhtap-service-push/rhtap-service-push.yaml

```diff
@@ -279,24 +301,31 @@ spec:
       runAfter:
         - push-sbom-to-pyxis
   finally:
-    - name: slack-webhook-notification
+    - name: send-slack-notification
       taskRef:
         resolver: "git"
         params:
           - name: url
-            value: https://github.com/redhat-appstudio/release-service-catalog.git
+            value: https://github.com/scoheb/release-service-catalog.git
```

Comment:
Why pointing to your fork when you are adding here the task?

Reply:
so I can test the PR from a RPA before merging.

I will change the location before merge.